src/core/algorithms/gfd/gfd.h

```diff
@@ -43,5 +43,9 @@ class Gfd {
         conclusion_ = conclusion;
     }
 

```

Comment:
Style Guide forbids name aliases in header files
https://google.github.io/styleguide/cppguide.html#Aliases:~:text=Don%27t%20put%20an%20alias%20in%20your%20public%20API%20just%20to%20save%20typing%20in%20the%20implementation%3B
```
7    using Token = std::pair<int, std::string>;
8    using Literal = std::pair<Token, Token>;
```